### ISSUE TITLE
How about we create a proxy_exporter?

### ISSUE THREAD

**Issue by mxlxm** [prometheus-community/community#35]
since almost all of the metrics exposed via http(s), how about we create a proxy_exporter which supports all kinds of `service` if we want. 
just wrap the other exporter, and call them via `service=xx` arg. 

here's my previous repo: https://github.com/mxlxm/exporters

**Comment by frittentheke** [comment]
The general idea sound like something https://github.com/QubitProducts/exporter_exporter does.

But you also want to include the actual metric extraction from the services into this proxy_exporter?
This would create one giant exporter having built-in support for lots of different services?

**Comment by mxlxm** [comment]
Yes, with this proxy_exporter we can scrape all kinds of services if we need. And we don't need to care about lots of different exporter's liveness.
